imap_processing/tests/swapi/test_swapi_decom.py

```diff
@@ -79,15 +74,26 @@ def test_swapi_hk_data(decom_test_data, swapi_l0_validation_data_path):
     """This test and validate raw data of SWAPI raw housekeeping data."""
     # read validation data
     raw_validation_data = pd.read_csv(
-        swapi_l0_validation_data_path / "idle_export_raw.SWP_HK_20231012_125245.csv",
+        swapi_l0_validation_data_path / "idle_export_raw.SWP_HK_20240924_080204.csv",
         index_col="SHCOARSE",
     )
 
     grouped_data = group_by_apid(decom_test_data)
     hk_packets = grouped_data[SWAPIAPID.SWP_HK]
     first_data = hk_packets[0]
     validation_data = raw_validation_data.loc[first_data.data["SHCOARSE"].raw_value]
-
+    bad_keys = [
```

Comment:
Are these bad in the validation data? I see you have a TODO on ln 112, is that still accurate?

Reply:
Yes. I just updated them. I plan to ask for new data but it's low priority.